privacyidea/static/components/config/views/config.privacyideaserver.list.html

```diff
@@ -11,7 +11,8 @@
         <tbody>
         <tr ng-repeat="(identifier, server) in privacyideaServers">
             <td>
-                <a ui-sref="config.privacyideaserver.edit({identifier:identifier})">
+                <a ui-sref="config.privacyideaserver.edit({identifier:identifier})"
+                   ng-click="editPrivacyideaServer(identifier)">
```

Comment:
So a state change always triggers a new request against the privacyIDEA server (probably like always instantiating a new controller :-))?
I guess our data layout is not really suited for this ui-router state machine logic...